cpp/include/cudf/hashing.hpp

```diff
@@ -92,6 +92,25 @@ std::unique_ptr<column> murmur_hash3_32(
   rmm::cuda_stream_view stream        = cudf::get_default_stream(),
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Computes the hash values of each row in the input set of columns
+ *
+ * This function takes a 64-bit seed value and returns hash values using the
+ * MurmurHash3_x64_128 algorithm. The hash produces in two uint64 values per row.
+ *
+ * @param input The table of columns to hash
+ * @param seed Optional seed value to use for the hash function
+ * @param stream CUDA stream used for device memory operations and kernel launches
+ * @param mr Device memory resource used to allocate the returned column's device memory
+ *
+ * @returns A table of two UINT64 columns
+ */
+std::unique_ptr<table> murmur_hash3_64_128(
```

Comment:
My preference would be getting rid of `_64_` in the name to align with the `hashing::detail::murmur_hash3_32` naming convention.

And TBH, it really should be `murmurhash3_32` (or `murmurhash_3_32`) since murmurhash is the name and 3 is the version number.

Out of curiosity, any reason we choose the `x64` version instead of `x86` one? I feel the latter is more often used.

Reply:
The 64 was introduced in case we want to add the 86 one later. I've no preference on the `x` and will wait for other feedback.
As per the choice of hash algorithm, I will attach a link to the previous comment where you asked why this was chosen.
https://github.com/rapidsai/cudf/pull/13604#issuecomment-1603257083